internal/attestation/renderer/chainloop/testdata/attestation.output.v0.2.json

```diff
@@ -7,11 +7,15 @@
         "sha256": "b403d1bb2b7739495df20f6d3838956333b04a66bc81d43bc7c64cee1433b604"
       }                                                                        
     },                                                                         
-    {                                                                                                                                                         
-      "name": "index.docker.io/bitnami/nginx",                                
-      "digest": {                                                              
+    {
+      "name": "index.docker.io/bitnami/nginx",
+      "digest": {
         "sha256": "580ac09da7771920dfd0c214964e7bfe4c27903bcbe075769a4044a67c9a390a"
-      }                                                                        
+      },
+      "annotations": {
```

Comment:
[not blocking / just a consideration]
Is it expected to have a JSON key in plural which value is not an array? In this case, it's `annotations`, meaning that 1 annotation is <key, value> in that hash.

Reply:
Yes, it's part of the [in-toto spec](https://github.com/in-toto/attestation/blob/main/spec/v1/resource_descriptor.md#schema) upstream.